addons/point_of_sale/static/src/js/models.js

```diff
@@ -1757,6 +1760,12 @@ exports.Orderline = Backbone.Model.extend({
             "taxDetails": taxdetail,
         };
     },
+    display_discount_policy: function(){
+        return this.product.get_discount_policy(this.order.pricelist);
```

Comment:
I would remove the `get_discount_policy` function on product and immediately return the value here. Currently the `get_discount_policy` on product doesn't actually use `this` so it's a bit weird to have it on that class.